packages/kas/src/__tests__/units.test.ts

```diff
@@ -44,33 +53,51 @@ expect.extend({
         );
         const answer = equality.solveLinearEquationForVariable(x);
         return Math.round(answer.eval()) === Math.round(expected.eval())
-            ? {pass: true}
+            ? {pass: true, message: () => ""}
             : {
                   pass: false,
                   message: () =>
                       `${original} = ["${expected.print()}"] ${newUnit}`,
               };
     },
-    toHaveTheSameForm([x, y]: [any, any], msg: string) {
+    toHaveTheSameForm(
+        [x, y]: [any, any],
+        msg: string,
+    ): jest.CustomMatcherResult {
         const equal = KAS.compare(
             KAS.unitParse(x).unit,
             KAS.unitParse(y).unit,
         ).equal;
 
-        return equal ? {pass: true} : {pass: false, message: () => msg};
+        return equal
+            ? {pass: true, message: () => ""}
+            : {pass: false, message: () => msg};
     },
-    toHaveMagnitude(input: string, expected: number) {
+    toHaveMagnitude(input: string, expected: number): jest.CustomMatcherResult {
         const parsed = KAS.unitParse(input).coefficient;
 
         return +parsed === expected
-            ? {pass: true}
+            ? {pass: true, message: () => ""}
             : {
                   pass: false,
                   message: () => `magnitude of ${input} is not ${expected}`,
               };
     },
 });
 
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace jest {
```

Comment:
Having to (re)declare these in each file we use them is a bit saddening. Hopefully we can figure out a way eventually. :)